src/ui/dfn-list.js

```diff
@@ -49,9 +49,34 @@ function show() {
           <a href="${`#${dfn.id}`}">
             ${dfn.textContent}
           </a>
+          ${labelDfnIfExported(dfn)} ${labelDfnIfUnused(dfn)}
         </li>
       `;
     });
   render`${definitionLinks}`;
   ui.freshModal(l10n.list_of_definitions, ul, button);
 }
+
+/**
+ * If a definition is exported, label it accordingly
+ * @param {HTMLElement} dfn a definition
+ */
+function labelDfnIfExported(dfn) {
+  if (dfn.hasAttribute("data-export")) {
+    return hyperHTML`(exported)`;
```

Comment:
So we can style this.
```suggestion
    return hyperHTML`<span class="exported">(exported)</span>`;
```